grepos/app/src/main/java/net/boondockradio/grepos/dto/Repositories.java

```diff
@@ -0,0 +1,18 @@
+
+package net.boondockradio.grepos.dto;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Repositories {
+
+    @SerializedName("total_count")
+    public int totalCount;
+    @SerializedName("incomplete_results")
+    public boolean incompleteResults;
+    @SerializedName("items")
```

Comment:
original name and `serializedName` is the same; in this case you don't need to declare `@SerializedName`. (If it was intentional it is fine but not necessary.)